packages/stencil-components/.storybook/stories/welcome/logo.svg

```diff
@@ -0,0 +1,12 @@
+<svg version="1.1" id="OktaLogo" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 431.9 151.4" style="enable-background:new 0 0 431.9 151.4;" xml:space="preserve">
```

Comment:
Would love to have an official Odyssey icon that we can use here!

Reply:
@joejezowski-okta is working on visuals for the new docs site, I suspect that we will have something to put here coming out of that!